transformations/summarization_transformation/transformation.py

```diff
@@ -0,0 +1,210 @@
+
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+import spacy
+
+"""
+The code is partially adapted and modified from https://github.com/NSchrading/intro-spacy-nlp
+"""
+
+
+class Summarization(SentenceOperation):
+    tasks = [
+        TaskType.TEXT_CLASSIFICATION,
+        TaskType.TEXT_TO_TEXT_GENERATION,
+    ]
+    languages = ["en"]
```

Comment:
Okay, one last thing - please add the keywords too.

Reply:
Thanks for this! I have added the keywords in the script.